README.md

```diff
@@ -1,93 +1,111 @@
 # Labs UI
-## NYC Planning Labs Ember Components
-This Ember addon contains common components, styles/CSS, Sass variables, utility classes, and design patterns that [NYC Planning Labs](https://planninglabs.nyc/) uses across its various applications.
+
+## NYC Planning Labs Ember.js Components
+
+Labs UI is an Ember.js addon containing common components and design patterns used across [NYC Planning Labs](https://planninglabs.nyc/) digital products, incorporating styles from our [Digital Style Guide](https://nyc-planning-style-guide.netlify.com/).
+
 
 ---
 
-### Documentation (using the addon)
+### Component documentation
```

Comment:
Love the changes in the headings on the README -- think it makes it easier to digest and get started.